src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -112,4 +114,11 @@ class JesBackendCall(val backend: JesBackend,
 
   override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
     backend.useCachedCall(avoidedTo.asInstanceOf[JesBackendCall], this)
+
+  override val pollBackoff = new ExponentialBackOff.Builder()
+    .setInitialIntervalMillis(30.seconds.toMillis.toInt)
+    .setMaxIntervalMillis(2.minutes.toMillis.toInt)
```

Comment:
Sounds a bit overkill to me too ATM, let's go with 3' start, up to 10' for now and see how it goes, if no one objects.